docs/schema/index-management/index-lifecycle.md

```diff
@@ -26,28 +26,72 @@ mgmt.commit();
+**REGISTERED**
 
--   **DISABLED** The index is disabled and no longer in use
+:   The index is known to all instances in the cluster.
+    However, it is not in use yet.
+    In this state, the index contains no data and is not eligible to be used in queries.
 
-## Actions (SchemaAction)
-The following actions can be performed on an index to change its state via `mgmt.updateIndex()`:
+**ENABLED**
+
+:   The index is enabled and can be used to answer queries.
+    Furthermore, this is the only state in which insertions and deletions in the graph are forwarded to the index.
 
--   **REGISTER_INDEX** Registers the index with all instances in the graph cluster. After an index is installed, it must be registered with all graph instances
+**DISABLED**
 
--   **REINDEX** Re-builds the index from the graph
+:   The index is disabled and will not be used to answer queries.
+It also does no longer receive graph updates.
+Therefore, it is recommended to use `REINDEX` to re-enable the index.
 
--   **ENABLE_INDEX** Enables the index so that it can be used by the query processing engine. An index must be registered before it can be enabled.
+**DISCARDED**
 
--   **DISABLE_INDEX** Disables the index in the graph so that it is no longer used.
+:   The index has been decommissioned and its contents have been deleted.
+    The only remaining artifact is the schema vertex representing the index.
+
+## Actions (SchemaAction)
+The following actions can be performed on an index to change its state via `mgmt.updateIndex()`:
 
--   **REMOVE_INDEX** Removes the index from the graph (optional operation). Only on composite index.
+**REGISTER_INDEX**
+:   Registers the index with all instances in the graph cluster.
+After an index is installed, it must be registered with all graph instances.
+
+**REINDEX**
+:   Re-builds the index from the ground up using the data stored in the graph.
+Depending on the size of the graph, this action can take a long time to complete.
+Reindexing is possible in all stable states and automatically enables the index once finished.
+
+**ENABLE_INDEX**
+:   Enables the index so that it can be used by the query processing engine.
+An index must be registered before it can be enabled.
+If enabling the index manually instead of performing a reindex, be aware that past modifications of the graph are not represented in the index.
+Enabling a previously disabled index without `REINDEX` may cause index queries to return ghost vertices, which have been deleted from the graph while the index was disabled.
+
+**DISABLE_INDEX**
+:   Disables the index temporarily so that it is no longer used to answer queries.
+The index also stops receiving updates in the graph.
+
+**DISCARD_INDEX**
+:   Removes all data from the index, preparing it for removal.
+    This operation is supported for all composite indices and for some mixed index backends.
+    For more information on index removal see [Removal](index-removal.md).
+
+**MARK_DISCARDED**
```

Comment:
Two questions:
1. Why is this action marked grey?
2. What happens if the user doesn't delete the real indices from the index backend manually, but still runs this action?

Reply:
Let me answer 2. first:
If the user performs this step without deleting the indexed data, all of it remains in the index backend of course. At the same time, it looks to JanusGraph as there this index does not exist and may never have. Therefore, JanusGraph would happily allow creating a new index with the same name that already exists in the backend. Even though I have not yet bothered to find out exactly __how__ things misbehave, I still consider this highly unsafe.

Now to 1.:
Grey means that an operation has to be used with care and may carry unforeseen consequences. In the case of this action, the user must be aware of the implications I explained in the previous answer (2)

Reply:
That makes sense. Would be even better if you could add a note/warning that one should not do `MARK_DISCARDED` without physically removing the indices.